### ISSUE TITLE
Packet Resend Failed

### ISSUE THREAD

**Issue by slushpuppy** [TeaSpeak/TeaSpeak#656]
Teaspeak version: 1.4.10

User disconnects with "Packet resend failed"

Users are able to reconnect immediately after

Logs:

[2020-03-26 05:10:42] [DEBUG]    18 | [--- | 15] Failed to execute packet resend: Failed to receive acknowledge for packet 143 of type Command
[2020-03-26 05:10:42] [DEBUG]    18 | [--- | 15] Client move timings: timings for : 0ms
[2020-03-26 05:10:42] [DEBUG]   - chan tree l: @0ms: 0ms
[2020-03-26 05:10:42] [DEBUG]   - channel res: @0ms: 0ms
[2020-03-26 05:10:42] [DEBUG]   - target show: @0ms: 0ms
[2020-03-26 05:10:42] [DEBUG]   - notify view: @0ms: 0ms
[2020-03-26 05:10:42] [DEBUG]   - lock own tr: @0ms: 0ms
[2020-03-26 05:10:42] [DEBUG]   - src chan up: @0ms: 0ms
[2020-03-26 05:10:42] [DEBUG]   - notify cpro: @0ms: 0ms
[2020-03-26 05:10:42] [INFO ] QUERY | [[0|X.X.X.X127.0.0.1:42349|unconnected]] Dropping new query connection attempt because of too many simultaneously connected session from this ip.
[2020-03-26 05:10:43] [DEBUG]    18 | [--- | 15] Failed to execute packet resend: Failed to receive acknowledge for packet 143 of type Command

Thanks again, may I know how I can assist with fixing this?

**Comment by WolverinDEV** [comment]
May try to update to the lasted beta. I've changed there something in that relation.

**Comment by slushpuppy** [comment]
[2020-04-04 01:17:41] [DEBUG]    18 | [/ | 26] Failed to execute packet resend: Failed to receive acknowledge for packet 176 of type Command

Latest beta: 1.4.11-beta-7

This seems to be happening to users with the latest 2 teamspeak versions

thanks

**Comment by WolverinDEV** [comment]
Are the users having a stable ping?
Because this could also just indicate an quite unstable connection.

**Comment by slushpuppy** [comment]
> 
> 
> Are the users having a stable ping?
> Because this could also just indicate an quite unstable connection.

I'll check, but all the users reconnect instantly after

**Comment by slushpuppy** [comment]
is there a way to increase timeout limit?

**Comment by slushpuppy** [comment]
u are right, but this has been happening to a lot of users:

![image](https://user-images.githubusercontent.com/1729407/78454627-c6903680-76cb-11ea-985e-b88b3444dd52.png)

How to trace and debug this?

**Comment by WolverinDEV** [comment]
Do you may have more than 32 users in one channel?

**Comment by REDMONDS** [comment]
> Do you may have more than 32 users in one channel?

Yes, it is possible... if the client lost the connection and tried to restore it and if there are more than 32 clients in the channel)
I think this is one of the cases, but I would also like to hear the answer)

**Comment by WolverinDEV** [comment]
This may has been increased in 1.4.12-beta4 ;)

**Comment by Crosus97** [comment]
Same problem here

if u want I can Drop my log if its necesary

**Comment by slushpuppy** [comment]
> 
> 
> Do you may have more than 32 users in one channel?

sometimes yes, majority of the time; 15ish ppl
